src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/ActionType.kt

```diff
@@ -42,6 +42,7 @@ data class ActionType<out T>(
 @JvmField val PROJECT_SUMMARY = ActionType("Project Summary") { Unit }
 @JvmField val SEARCH_WITH_JQL = ActionType("Search with JQL") { SearchJqlObservation(it) }
 @JvmField val SEARCH_WITH_JQL_A_STAR = ActionType("Search with JQL a*") { SearchJqlObservation(it) }
```

Comment:
Maybe it makes sense to rename this to `SEARCH_WITH_JQL_WITH_WILDCARD` to be consistent?

Reply:
Or maybe `SEARCH_WITH_JQL_WILDCARD`

Reply:
Hopefully we have not released the change yet because this would be API breaking.

Reply:
We haven't released it yet, that's why we can still change it